Resources/Prototypes/_Impstation/Catalog/Cargo/cargo_armory.yml

```diff
@@ -16,4 +16,14 @@
   product: CrateArmoryBurstPistol
   cost: 7000
   category: cargoproduct-category-name-armory
+  group: market
+
+- type: cargoProduct
+  id: ArmoryDMR
+  icon:
+    sprite: _Impstation/Objects/Weapons/Guns/Rifles/daito.rsi
+    state: icon
+  product: CrateArmoryDMR
+  cost: 10000
```

Comment:
10k might be a bit expensive considering the wt550s are 9k, but i suppose it's better to err on the side of caution without having experienced these guns in a real environment